File: EFTEDashboard/src/app/interfaces/relationships/rerolled-card-card-instance.ts
import { CardChoiceRecord } from "../card-choice-record";
import { CardInstance } from "../Instances/card-instance";

export interface RerolledCardCardInstance {
  cardChoiceRecordId?: number;
  cardChoiceRecord?: CardChoiceRecord;
  cardInstanceId?: number;
  cardInstance?: CardInstance;
}

File: EscapeFromTrinityEngineStats/Models/Relationships/RerolledCardCardInstance.cs
using EscapeFromTrinityEngineStats.Models.Instances;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EscapeFromTrinityEngineStats.Models.Relationships
{
    public class RerolledCardCardInstance
    {
        public int CardChoiceRecordId { get; set; }
        public int CardInstanceId { get; set; }
        public virtual CardInstance CardInstance { get; set; }
    }
}
